soroban-env-host/src/host/crypto.rs

```diff
@@ -106,6 +229,7 @@ impl Host {
         }
     }
 
+    #[cfg(not(feature = "next"))]
```

Comment:
I don't think we can compile this out (unless no-one ever uses it until p21)

Reply:
See comment above. There is a generic version `ecdsa_signature_from_bytesobj_input<C>` replacing this one for p21.